composer.json

```diff
@@ -30,17 +30,17 @@
         "friendsofsymfony/http-cache": "^2.5",
         "friendsofsymfony/http-cache-bundle": "^2.6",
         "friendsofsymfony/jsrouting-bundle": "^2.3",
-        "friendsofsymfony/rest-bundle": "^2.4",
+        "friendsofsymfony/rest-bundle": "^2.5",
         "gedmo/doctrine-extensions": "^2.4",
         "goodby/csv": "^1.3",
         "guzzlehttp/guzzle": "^6.2",
         "guzzlehttp/psr7": "^1.5",
         "imagine/imagine": "^0.6.1 || ^0.7.0 || ^1.0",
-        "jms/serializer": "^1.13",
-        "jms/serializer-bundle": "^2.3",
+        "jms/serializer": "^3.0",
+        "jms/serializer-bundle": "^3.0",
         "layershifter/tld-extract": "^1.1 || ^2.0",
         "massive/build-bundle": "^0.3 || ^0.4",
-        "massive/search-bundle": "^0.16 || ^0.17 || ^1.0",
+        "massive/search-bundle": "dev-feature/dependency-update",
```

Comment:
Need to be merged and tagged as 2.0 before this PR can be merged.
https://github.com/massiveart/MassiveSearchBundle/pull/132